mh-better-spooky-shuffle-tracker.user.js

```diff
@@ -72,12 +72,21 @@
 		return JSON.parse(localStorage.getItem('mh-spooky-shuffle-cards')) || [];
 	};
 
-	const renderSavedCard = (card) => {
-		if (! card) {
-			return;
+	const getSavedBoards = () => {
+		return JSON.parse(localStorage.getItem('mh-spooky-shuffle-boards')) || {};
+	};
+
+	const isNewBoard = (board, req) => {
+		// Check if all of the card names are null.
+		if (board.every((card) => card.name === null)) {
+			return true;
 		}
 
-		if (card.is_matched) {
+		return (req.memory_game.num_tickets !== localStorage.getItem('mh-spooky-shuffle-cached-tickets'));
```

Comment:
oh actually, this was one of the lint suggestions that breaks things as these 2 aren't equivalent down to type. Is what they want me to do to force the type on one/both so that they match for this equality check?

Reply:
Ahh, yeah - I didn't test this actually.

I'd say revert to what you had, but maybe with a comment saying that.

Reply:
really showing how much of not-a-dev I am, how do I revert it if it's your PR? 

I was planning on doing all the fixes I need for functionality after you merge this for formatting